handler.go

```diff
@@ -28,14 +24,16 @@ func (v *verifier) handleCheckImages(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(requestData.Images) == 0 {
+	if len(requestData.Containers) == 0 &&
+		len(requestData.InitContainers) == 0 &&
+		len(requestData.EphemeralContainers) == 0 {
 		log.Printf("missing images in %v", requestData)
```

Comment:
Here we should return an empty "200 OK" response as the format, etc. is correct.